specification/applicationinsights/resource-manager/readme.python.md

```diff
@@ -1,27 +1,110 @@
 ## Python
 
+## python
 These settings apply only when `--python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
 
-``` yaml $(python)
-python-mode: create
```

Comment:
to remove python-mode: create is just a temporary solution for az. I suggest you hold on.

Reply:
Just for SDK, @MyronFanQiu please revert this change

Reply:
Done. Please check other part. You can directly modify these files since I just got this file from wei dong. I don't have full knowledge as well.